lang/LangPrimSource/PyrPrimitive.cpp

```diff
@@ -3761,9 +3761,9 @@ void doPrimitive(VMGlobals* g, PyrMethod* meth, int numArgsPushed)
 
 	if (diff != 0) { // incorrect num of args
 		if (diff > 0) {  // not enough args
-			PyrSlot* pslot = g->sp;
-			PyrSlot* qslot = slotRawObject(&meth->prototypeFrame)->slots + numArgsPushed - 1;
-			for (int m=0; m<diff; ++m) slotCopy(++pslot, ++qslot);
+			PyrSlot* pslot = g->sp + 1;
```

Comment:
compare https://github.com/supercollider/supercollider/pull/2115/files#diff-4cbdf08b4b73d0efb6809e4667175290R312

it prevents a read-after-write data hazard, as in the unrolled part, we first read 4 slots and then write 4 slots instead of interleaving reads and writes. originally the read operation had to wait for the write operation to complete.